src/book/garden-circle/contributing.md

```diff
@@ -137,6 +137,27 @@ These workflows include one or more [jobs](https://docs.github.com/en/actions/us
 We define GitHub Actions work under the [`.github`](https://github.com/software-gardening/almanac/tree/main/.github) directory.
 We suggest the use of [`act`](https://github.com/nektos/act) to help test GitHub Actions work during development.
 
+### Releases
+
+We utilize [semantic versioning](https://en.wikipedia.org/wiki/Software_versioning#Semantic_versioning) ("semver") in order to distinguish between major, minor, and patch releases.
```

Comment:
```suggestion
We utilize [semantic versioning](https://en.wikipedia.org/wiki/Software_versioning#Semantic_versioning) ("semver") to distinguish between major, minor, and patch releases.
```

the phrase "in order to" is almost always fluff that can be easily removed :)

Reply:
Thanks!